README.md

```diff
@@ -29,7 +29,9 @@ module system it is exported as `GraphQLVoyager` global variable.
 ### Properties
 `Voyager` component accepts the following properties:
 
-+ `introspection` [`object` or function: `(query: string) => Promise`] - the server introspection response. If `function` is provided GraphQL Voyager will pass introspection query as a first function parameter. Function should return `Promise` which resolves to introspection response object.
++ `fetcher` _(optional)_ [function: `(query: string) => Promise`] - a function for fetching an introspection of your schema. GraphQL Voyager will pass an introspection query as a first function parameter. Function should return `Promise` which resolves to introspection response object.
```

Comment:
GraphiQL pass an object with `query` and `operationName` (also possible `variables`) so you send it directly to the server:
https://github.com/graphql/graphiql/blob/master/packages/graphiql/src/components/GraphiQL.js#L580

Reply:
We should do the same so you can use same function for both Voyager and GraphiQL